infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/LogReplicationEvent.java

```diff
@@ -57,7 +57,9 @@ public enum LogReplicationEventType {
      * @param type log replication event type
      */
     public LogReplicationEvent(LogReplicationEventType type) {
-        this(type, LogReplicationEventMetadata.empty());
```

Comment:
I don't think we had any other usages of LogReplicationEventMetadata.empty() so we might be able to remove that

Reply:
yes, the line was removed...or are you pointing at something else?

Reply:
Sorry, I mean't the implementation of empty() in LogReplicationEventMetadata.java wasn't able to comment on non-edited lines.